src/index.html

```diff
@@ -7,7 +7,56 @@
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+  
   <body>
-    <h1>Stars</h1>
+    <div class="all-stars">
+      <div class="stars stars--0">
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </div>
```

Comment:
would be great to use semantic tags here. You got a list with list items here, right?

Reply:
You've recommended change <div> on ul>li*5, Am I clearly understanding you?